### ISSUE TITLE
Does not set JAVA_HOME

### ISSUE THREAD

**Issue by scripthead** [softek/puppet-java7#2]
After installation JAVA_HOME is empty. This probably should add

JAVA_HOME=/usr/lib/jvm/java-7-oracle

into /etc/environment

**Comment by smerchek** [comment]
I don't believe it is the responsibility of this module to set `JAVA_HOME` in a global area. I would expect this of the `oracle-java-7` package, if this was indeed desired.

As far as I can tell, there isn't a java package that does this (I also checked `openjdk`), so I don't think this is a valid issue.

However, I could see that it might be helpful to expose a puppet variable on the `java7` class, so that other modules know where `JAVA_HOME` is.

**Comment by mcandre** [comment]
+1 Puppet's supposed to help us automate setting up these nitty gritty details.

**Comment by smerchek** [comment]
Note: It appears that the package `oracle-java7-set-default` will set JAVA_HOME. This might be the best way of handling it in the puppet module.
